src/games/strategy/sound/SoundProperties.java

```diff
@@ -27,7 +31,10 @@ protected SoundProperties(final ResourceLoader loader) {
       // no property file found
     } else {
       try {
-        m_properties.load(url.openStream());
+        Optional<InputStream> inputStream = UrlStreams.openStream(url);
+        if(inputStream.isPresent()) {
+          m_properties.load(inputStream.get());
+        }
       } catch (final IOException e) {
```

Comment:
`m_properties.load(inputStream.get());` throws IOException, so we still have to catch it.